docs/reference/elastic-search.md

```diff
@@ -106,3 +109,48 @@ WIRE_VERSION=<VERSION_YOU_ARE_DEPLOYING>
 1. Redeploy brig without `elasticsearch.additionalWriteIndex` and with `elasticsearch.index` set to the name of new index
 
 Now you can delete the old index.
+
+## Recreate an index
+
+When analysis settings of an index need to be changed, e.g. for changes
+introduced in [#1052](https://github.com/wireapp/wire-server/pull/1052),
+it is not possible to keep the index running while the changes are applied.
+
+To tackle this, a wire-server operator must either migrate to a new index as
+documented [above](#migrate-to-a-new-index) or allow for some downtime. One
+might want to choose downtime for simplicity. These steps are especially simple
+to do when using [wire-server-deploy](https://github.com/wireapp/wire-server-deploy/).
+
+Here are the steps:
+
+Before starting, please set these environment variables
```

Comment:
Can we separate the steps a bit more clearly? As in:

# First step

`1. Delete the old index and the migrations index`

# Second step

## If you use helm charts....

## If you do _not_ use helm charts....

WDYT?

Reply:
Hmm, good idea, I wiil try it out to see how it looks.

Reply:
Looks much better, thanks!